### ISSUE TITLE
[Q] failing to understand emojis instructions

### ISSUE THREAD

**Issue by gcb** [mintty/mintty#1058]
Trying to follow the wiki instructions.

Right out of the box I do get windows outlined emoji support, which is already great, but for some reason i want to try to have color ones, so I am following the wiki for it.

created `~/.config/mintty/emojis`, and cd into.

run `getemojis -d apple` (later i tried everything again with the windows set, same end result)

I get:
```
 $ ls -1
0/
1/
apple/
common/
full-emoji-list.html
full-emoji-modifiers.html
```

After all is done, i quit and restart mintty. In the Options, Text dialog, I do see "apple" (later also "windows") showing in the emoji drop-down. But that's it. Changing it does nothing.

tried to add to `~/.config/mintty/config`:
```
Charwidth=unicode
Emojis=apple
```

Tried changing fonts too. All I can get is the B&W outline emojis. What am I doing wrong?

**Comment by gcb** [comment]
Just now i noticed the common/apple/windows directories are empty, and all images are in 0/1/4 ones. Is this expected?

**Comment by mintty** [comment]
That is not expected and you followed the instructions properly, also it's not necessary to configure emojis manually in the config file and not even to restart mintty.
In your ls output, final "/" suggests that 0 and 1 are not symbolic links as they should. Is that on a FAT filesystem (USB stick perhaps)?
Please try a workaround: `mv 0/* common/; mv 1/* apple/`.
I'll try to change the script to work on FAT too.

**Comment by gcb** [comment]
Thank you! (for the quick help and making life on windows bearable :)

My setup is to just install git-for-windows. And I give up some convenience to avoid problems, such as not enabling symlinks on NTFS

Your workaround worked nicely! moved 0->common, 1->apple, 4->windows. Didn't change anything immediately, so i tried a restart again. i was restarting mostly because i saw the code to look for the emojis in `config.c` (though i'm not familar with the codebase, the further i managed to follow it seems it was just populating the dropdown, which was OK even before). This time it did the trick! now changing on the options between apple/windows i see the changes immediately.

Thanks again! :+1

**Comment by mintty** [comment]
Hmm, indeed MSYS2 (which Git-on-Windows is based on) does not seem to support symbolic links, weird, I wasn't aware.

**Comment by Biswa96** [comment]
MSYS2 is a fork of cygwin. Everything that cygwin does, MSYS2 can.

**Comment by k-takata** [comment]
However, `ln -s` just creates a copy of the specified file by default on MSYS2.

**Comment by Biswa96** [comment]
`export MSYS=winsymlinks:nativestrict` <--> `export CYGWIN=winsymlinks:nativestrict` [doc][1]

[1]: https://cygwin.com/cygwin-ug-net/using-cygwinenv.html

**Comment by mintty** [comment]
Tweaked the script and uploaded to wiki. Testing welcome.

**Comment by mintty** [comment]
Released 3.4.3.
